Refactor/typesafe content

seinjonetan opened the pull request:
Updated the json files to ts files. With the purpose to make the json objects typesafe.

seinjonetan committed:
fix: made studies content typesafe
(kept original json file)

seinjonetan committed:
fix: made faqs typesafe
(kept original json file in contents)

seinjonetan committed:
fix: made testimonials typesafe
(kept original json file in contents)

seinjonetan committed:
fix: made prices typesafe
(kept original json file )

vvel0x commented:
The data should not be changed - could you refactor to maintain the existing data? It might also be worth considering adding barrel exports.

seinjonetan committed:
fix: - refectored studies to maintain original
data structure
- added initial index.ts file in contents folder for barrel export

seinjonetan committed:
fix: implemented barrel exports for testimonials,
faq and prices in contents folder
